File: QViz.NET/Periscope.QViz/JSON/APIHeader.cs
using System;

namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for Test API Header Details
	/// </summary>
	public class APIHeader
	{
		/// <summary>
		/// Identifier of the API Header Object
		/// </summary>
		public string apiHeaderId;

		/// <summary>
		/// Serial Number of the API Header Object
		/// </summary>
		public int srNo;

		/// <summary>
		/// Identifier of the API Object
		/// </summary>
		public string apiId;

		/// <summary>
		/// Identifier of the API Header Value Object
		/// </summary>
		public string headerValueId;

		/// <summary>
		/// Date and Time on which this Model was Created
		/// </summary>
		public DateTime createdOn;

		/// <summary>
		/// Date and Time on which this Model was last Updated
		/// </summary>
		public DateTime updatedOn;

		/// <summary>
		/// Value of the API Header Object
		/// </summary>
		public APIHeaderValue headerValue;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public APIHeader()
		{
			this.apiHeaderId = "";
			this.srNo = 0;
			this.apiId = "";
			this.headerValueId = "";
			this.createdOn = DateTime.Now;
			this.updatedOn = DateTime.Now;
			this.headerValue = new APIHeaderValue();
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/APIHeader.java
package com.periscope.qviz.json;

import java.util.Date;

/**
 * JSON Model for Test API Header Details
 */
public class APIHeader {

	/**
	 * Identifier of the API Header Object
	 */
	public String apiHeaderId;

	/**
	 * Serial Number of the API Header Object
	 */
	public Integer srNo;

	/**
	 * Identifier of the API Object
	 */
	public String apiId;

	/**
	 * Identifier of the API Header Value Object
	 */
	public String headerValueId;

	/**
	 * Date and Time on which this Model was Created
	 */
	public Date createdOn;

	/**
	 * Date and Time on which this Model was last Updated
	 */
	public Date updatedOn;

	/**
	 * Value of the API Header Object
	 */
	public APIHeaderValue headerValue;

	/**
	 * Default Constructor
	 */
	public APIHeader() {
		this.apiHeaderId = "";
		this.srNo = 0;
		this.apiId = "";
		this.headerValueId = "";
		this.createdOn = null;
		this.updatedOn = null;
		this.headerValue = new APIHeaderValue();
	}

}
